src/posts/xrisk.md

```diff
@@ -5,11 +5,9 @@ description: Why AI is a risk for the future of our existence, and why we need t
 
 ## Experts are sounding the alarm
 
-AI researchers on average [believe](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/) there's a 14% chance that once we build a superintelligent AI, it will lead to "very bad outcomes (e.g. human extinction)".
+AI researchers on average [believe](https://aiimpacts.org/2022-expert-survey-on-progress-in-ai/) there's a 5% chance that once we build a superintelligent AI, it will lead to "very bad outcomes (e.g. human extinction)".
 
-Would you choose to be a passenger on a test flight of a new plane when airplane engineers think there’s a 14% chance that it will crash?
-
-When you ask AI safety researchers (the people who actually research the safety risks of AI), the average estimate for bad outcomes [grows to 30%](https://forum.effectivealtruism.org/posts/8CM9vZ2nnQsWJNsHx/existential-risk-from-ai-survey-results).
```

Comment:
Why leave this out? I think AI safety researchers are far more likely to have given this subject serious thought than an average AI researcher. Their estimates are valuable.

Reply:
See discussion above. I agree their estimates are more valuable. But only because I _already_ believe in AI x-risk.